templates/index.html

```diff
@@ -36,21 +17,22 @@
 
     <div class="meetup">
 
-      <h3 class='meetup-name'><a href="{{ meetup.meetup_link }}">{{ meetup.name }}</a></h3>
+      <h3 class='meetup-name'><a href="{{ meetup.get_meetup_link }}">{{ meetup.get_meetup_name }}</a></h3>
 
-      {% if meetup.next_event_topic %}
+      {% if meetup.get_meetup_next_event_topic %}
```

Comment:
Consider defining your presenter API by working back from the template syntax you'd like to use. (In SICP, this is called ["programming by wishful thinking"](http://c2.com/cgi/wiki?WishfulThinking)).

Here's an example of a nicer template syntax (this is off the top of my head...I'm not suggesting this is necessarily the right choice):

```html
{% if meetup.has_upcoming_event %}
  <div class='meetup-description'>
    <strong class='subheader'>Next Event:</strong>
    <p>{{ meetup.upcoming_event.topic }} — {{ meetup.upcoming_event.date_and_time }} at
    <a href="{{ meetup.upcoming_event.map_link }}">{{ meetup.upcoming_event.venue }}</a></p>
  </div>
{% endif %}
```

How would the presenter API need to change to support a syntax like this? Do you think it's more readable?